### ISSUE TITLE
Support for .NET 5.0

### ISSUE THREAD

**Issue by DominicSachs** [ivaylokenov/MyTested.AspNetCore.Mvc#383]
Is there any plan to support .NET 5?
At the moment an exception is thrown:  "This version of MyTested.AspNetCore.Mvc only supports ASP.NET Core 3.1 applications but the 5.0.0 web framework was referenced"

**Comment by ivaylokenov** [comment]
@DominicSachs Yes, it is currently under development. It should be released soon.

**Comment by wu-yafeng** [comment]
Any ETA or preview release available?

**Comment by chuac** [comment]
Hey @ivaylokenov, any update on the ETA?

**Comment by ivaylokenov** [comment]
Unfortunately, the project currently is a really low priority for me. Ever since the pandemic started almost everyone who was supporting My Tested ASP.NET is no longer doing it: https://opencollective.com/mytestedaspnet
Back in the day, the community donations allowed me to cover all the maintenance costs. Nowadays, I had to focus on other activities to generate income. For this reason, I will not be able to give an ETA. As soon as I find enough free time, I will upgrade the packages and release a new version.

**Comment by ivaylokenov** [comment]
Guys, version 5.0 is deployed to NuGet - it is completely free and does not require a license from now on. I still need to update some other text files in the repository, but you download it and use it for your tests.

**Comment by stefanAtUcommerce** [comment]
Hi Ivo, Is it free only with version 5.0?

**Comment by ivaylokenov** [comment]
@stefanAtUcommerce Version 3.1 also does not require a license code from now on. Version 6.0 will not require either but it will be released in a few days hopefully.

**Comment by stefanAtUcommerce** [comment]
@ivaylokenov Thank you, you are the best :)
